submissions/TetianaDiachenko/popup_td/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="wrapper">
+            <div class="header__content">
+                <div class="header__logo">
+                    <a href="https://kottans.org/" target="_blank">
+                        <img src="./img/kottans.png" alt="kottans">
+                    </a>
+                </div>
+                <nav class="header__nav">
+                    <ul class="header__list" >
+                        <li class="header__item">
+                            <a href="#">Mail</a>
```

Comment:
Avoid elements shifting on hover or focus stgate. [Watch video](https://share.vidyard.com/watch/pPNgFXoNLxepcK7MKUbWKx?)

Reply:
Thank you for you answer. 
If you could please say how to do this? or what to read/watch how to do this?

Reply:
1. You can use an outline or border
2. By default you can set a transparent border and just change border color when you need it
Also, I'm right now streaming in discord so  you can ask me dirrectly)